tools/bedtools_intersect.cwl

```diff
@@ -25,10 +25,10 @@ arguments:
             return "echo No vcf provided, skipping >&2 && exit 0;"
           }
           if (inputs.flag != "N"){
-            cmd += "bedtools intersect -a " + inputs.input_vcf.path + " -b " + inputs.input_bed_file.path + " -header -wa > " + out_vcf + " && ";
+            cmd += "bedtools intersect -a " + (inputs.input_vcf ? inputs.input_vcf.path : "") + " -b " + inputs.input_bed_file.path + " -header -wa > " + out_vcf + " && ";
```

Comment:
Hmmm, if `inputs.input_vcf` is null, then give empty string to `-a`? How does that work?

Reply:
This is to get around a "null has no attribute path" error. In reality if the input_vcf is not provided, we'll have exited anyway.